src/config.py

```diff
@@ -163,8 +204,12 @@ def load(cls, file_path: Path | str = "config.yaml") -> RuntimeConfig:
             ValueError: If the configuration contains invalid database types
 
         """
-        with open(file_path, encoding="utf-8") as _handle:
-            data = cls.read_yaml(_handle)
+        # TODO maybe support other schemes - what if it lives on an FTP server? :^)
+        #  alternatively, enforce scheme - files would be passed as "file:///path/to/config.yaml"
+        if re.match(r"^https?://.*", str(file_path)):
```

Comment:
Doesn't python have some library like parse url or is url so we can try parse url or assume path. Might be more robust than regex.

Reply:
Fair point, it's probably a better idea to use something in `urllib` that either returns a URL or None, I'll fix it.

Reply:
> How do our mocked tests ensure that the fetching actually works

Well the fetching is performed by `aiohttp` and I'd rather not test external code, but I agree that an end-to-end test would be beneficial. I'll fix one up.